server/pragma-checker/index.js

```diff
@@ -19,7 +19,9 @@ var IGNORED_MODULES = [
 	'lib/upgrades/actions', // nooped on the server as it still uses the singleton Flux architecture
 	'i18n-calypso', // ignore this until we make it work properly on the server
 	'my-sites/themes/thanks-modal', // stubbed on the server until we develop an isomorphic version
-	'my-sites/themes/themes-site-selector-modal' // stubbed on the server until we develop an isomorphic version
+	'my-sites/themes/themes-site-selector-modal', // stubbed on the server until we develop an isomorphic version
+	'state/ui/editor/selectors', // stubbed on the server until all the dependencies are @ssr-ready
+	'state/posts/selectors', // stubbed on the server until all the dependencies are @ssr-ready
```

Comment:
thanks @ehg - I stubbed them in another PR and will update this one here to do the same